File: cash.go
package main
import "fmt"


func main() {
    
    var f float64

         fmt.Println("Enter a float value : ")
         _, err := fmt.Scanf("%f", &f)

         if err != nil {
            fmt.Println("not number")
         }
         
    sum := f*100
    var counter int
    counter = 0
    	for sum >= 25 {
    		counter += 1
    		sum += -25
    	}
    	for sum >= 10 {
    		counter += 1
    		sum += -10
    	}
    	for sum >= 5 {
    		counter += 1
    		sum += -5
    	}
    	for sum >= 1 {
    		counter += 1
    		sum += -1
    	}
    	
    	fmt.Println(counter)

}

File: cash.cs
using System;
class HelloWorld {
  static void Main() {
    Console.WriteLine("Enter sum of money");
    float floatNumber = 0.0f;
    int intNumber;      
        // use of Single.Parse() Method
        floatNumber = float.Parse(Console.ReadLine());
    intNumber = (int)Math.Round(floatNumber*100);
    
    int i = intNumber;
    int counter = 0;
    while (i >= 25) 
    {
      i = i - 25;
      counter++;
    }
    
    while (i >= 10) 
    {
      i = i - 10;
      counter++;
    }
    
    while (i >= 5) 
    {
      i = i - 5;
      counter++;
    }
    
    while (i == 1) 
    {
      counter++;
      i = i - 1;
    }

    Console.WriteLine(counter);
  }
}
